projects/kserve/kserve-incubation-dd.md

```diff
@@ -0,0 +1,333 @@
+# KServe Incubation Due Diligence
+
+- Link to [Incubation application issue](https://github.com/cncf/toc/issues/1367)
+
+<!-- This template provides the TOC with the outline for completing due diligence of a project to move levels. This universal template is designed to capture all criteria so the TOC may ensure prior level criteria do not regress. As part of completing the due diligence, the TOC member should update the template to convey the level the project applied for the criteria by bolding the level indicated where the criteria is relevant. -->
+
+## Incubation Evaluation Summary for KServe
+
+### Criteria Evaluation
+
+_$TOCMEMBER conducted the due diligence of KServe who applied for $LEVEL. The project [has/has not] completed the criteria that show its maturity at $LEVEL. The following criteria implementations are noteworthy to call out... $NOTABLES. The following actions were provided to the project that were considered blocking but since resolved... $BLOCKERS. The following recommendations were provided to the project that are non-blocking in the TOC's assessment but should be completed by the project to ensure continued viability of the project... $RECOMMENDATIONS._
+
+### Adoption Evaluation
+
+_The adopter interviews reflect a project [in use/too early] for the level which the project applied. They show ... $INTERVIEWSUMMARY._
+
+### Final Assessment
+
+_[The TOC has found the project to have satisfied the criteria for $LEVEL/ The TOC's evaluation of the project shows a needed focus to complete the outstanding blockers and reapply when the following conditions are met ... $CONDITIONS]._
+
+## Application Process Principles
+
+### Suggested
+
+N/A
+
+### Required
+
+- [ ] **Give a presentation and engage with the domain specific TAG(s) to increase awareness**
+  - This was completed and occurred on DD-MMM-YYYY, and can be discovered at $LINK.
+
+  <!-- (TOC Evaluation goes here) -->  
+
+  **TODO for KServe team**: Provide GTR/DTR files
+
+- [ ] **TAG provides insight/recommendation of the project in the context of the landscape**
+
+  <!-- (TOC Evaluation goes here) -->  
+  **TODO for KServe team**: fill GTR/DTR
+
+  **TODO for Kevin**: check with TAG runtime to provide recommendation
+
+- [ ] **All project metadata and resources are [vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).**
+
+  <!-- (TOC Evaluation goes here) -->  
+  Yes. KServe is utilizing vendor neutral resources for communication, testing, hosting and governance.
+
+  **TODO for KServe team**: provide links do prove this.
```

Comment:
Tracking in https://github.com/kserve/community/issues/26

Reply:
@kevin-wangzefeng Please review https://github.com/kserve/community/issues/26#issuecomment-2866903229 and help add it to the DD doc in your fork.